### DISCUSSION TITLE
Accessing the backups in s3 from different longhorn instance/cluster?

### DISCUSSION THREAD

**Discussion by Szwendacz99** [longhorn/longhorn#9463] [Q&A] [answered]
Is this possible and supported to allow longhorn access s3 backups while this s3 path is actively being used by another longhorn instance in different k8s cluster? If not, what would be the best idea to migrate volumes to different cluster?

**Comment by shuo-wu** [comment, marked as answer]
Yes. One typical use case is:
Cluster A (Primary cluster) periodically backup its volumes into a remote backupstore, while Cluster B (disaster recovery(DR) cluster) creates DR volumes based on the backups in the remote backupstore and periodically fetches the latest data into the volumes. When the primary cluster A is down, users can directly activate the DR volumes in cluster B as normal volumes then immediately launch the services based on these volumes.
